packages/oas-to-har/__tests__/lib/style-formatting/index.test.js

```diff
@@ -0,0 +1,948 @@
+const Oas = require('oas/tooling');
+
+const oasToHar = require('../../../src/index');
+
+const oas = new Oas();
+
+const emptyInput = '';
+const undefinedInput = undefined;
+const stringInput = 'blue';
+const arrayInput = ['blue', 'black', 'brown'];
+const undefinedArrayInput = [undefined];
+const objectInput = { R: 100, G: 200, B: 150 };
```

Comment:
I'm afraid to ask but how does it handle nested objects, arrays of objects, or arrays of arrays?

Reply:
I'm having a hard time finding it, but somewhere said that nested objects or arrays are not allowed. I think it might be somewhere in the URI template spec that each style is from. Still looking

As for what would happen, I believe it would apply javascript stringification (via `String(val)`), so we would get something like `R=[object]`

Reply:
Works for me. I can't imagine anyone actually trying to use nested data in a URI template anyways.